src/client/datascience/baseJupyterSession.ts

```diff
@@ -63,6 +63,12 @@ export abstract class BaseJupyterSession implements IJupyterSession {
         }
         return this.onStatusChangedEvent.event;
     }
+    public get onIOPubMessageSignal(): Event<KernelMessage.IIOPubMessage> {
```

Comment:
Might want to remove the "Signal" from the name here, it's only a Signal down in the Juypter code doesn't really make sense to have that on the name here.